Gemfile

```diff
@@ -28,6 +28,7 @@ gem 'has_permalink'
 gem 'authlogic'
 gem 'scrypt' # for authlogic
 gem 'request_store' # for authlogic
+gem "paranoia", "~> 2.2"
```

Comment:
Can we remove the [pessimistic constraint](https://guides.rubygems.org/patterns/#declaring_dependencies) here? 

The current version is 2.4.x but I usually have a policy of not locking gems in the Gemfile unless there is known problems with upgrading to a higher version (otherwise things like `bundle update` and dependabot will never update it, we'd have to remember to remove it later)